### ISSUE TITLE
Refresh rate control for user

### ISSUE THREAD

**Issue by Gyoshi** [LiveSplit/LiveSplit.ScriptableAutoSplit#60]
A [control](https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.control) for letting the user change the refresh rate could be added to the settings of the component. This would allow the user to manually change split timing precision.

Seems like it would be problematic for load removers since refresh rate can actually affect a run's no-load time. And you wouldn't want it affecting different runners' times differently (could incentivise having really low refresh rate to artificially extend load times). Perhaps solvable by disabling (greying out) the control if the script incorporates `isLoading`.

Should probably also be capped by the livesplit timer's refresh rate. ([or a little faster](https://github.com/LiveSplit/LiveSplit.ScriptableAutoSplit/blob/f599ae9900b02fb1166e7fb6e47c35452cd74265/Component.cs#L49))

Would be nice if (like the other settings) it can be changed and use the new value even mid-run. Otherwise it should be disabled while the timer is running.

**Comment by Jujstme** [comment]
Other than LiveSplit's main update interval, each autosplitter can freely change its refreshRate as documented <a href=https://github.com/LiveSplit/LiveSplit.AutoSplitters#refreshrate>here</a>

Also coded here: https://github.com/LiveSplit/LiveSplit.ScriptableAutoSplit/blob/f599ae9900b02fb1166e7fb6e47c35452cd74265/Component.cs#L161

Keep in mind the default value is roughly 60 times per seconds, but its precision can vary due to an intrinsic limitation of Microsoft's implementation of System.Threading.Timer.

If a game needs a higher precision than what LiveSplit provides, probably it needs a different implementation altogether, like a timer integrated into the game itself.

**Comment by Gyoshi** [comment]
Yes I know this, I meant that the user of an autosplitter (not an author) could be able to adjust it, same as how the timer's refresh rate is adjustable for in LiveSplit's settings. I added a picture for clarity.

I was going to make a separate issue about the precision of the timer

**Comment by Gyoshi** [comment]
This feature is kind of pointless until https://github.com/LiveSplit/LiveSplit.ScriptableAutoSplit/issues/61 is fixed
